### PR TITLE
Issue 511: Raise expected proficiency level for the Secured Mission to Advanced + add callout to intro

### LINKED ISSUE #511: Change the expected proficiency level for the Secured mission to Advanced
The complexity of the SSO Secured mission warrants a higher expected proficiency level.

Reasons:
* The mission has multiple alternative modes of interaction (JAR client, `curl`, and web client).
* The web client requires Keycloak realm reconfiguration in order to be accessible, it is not accessible out of the box as is the case with the other missions currently in the catalog.
* The mission expects the user to know how to configure SSO.
* SSO configuration is not within the scope of the mission.

FYI @Ladicek

### PR DESCRIPTION
**Pull request by inoxx03**
resolves #511

### CODE REVIEW

**Comment by inoxx03** [comment]
@Ladicek WDYT?

File: docs/topics/secured-mission-intro-paragraph.adoc [hunk 1]
```diff
@@ -1,6 +1,8 @@
...
-ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
-ifndef::docs-topic[link:https://appdev.openshift.io/docs/wf-swarm-runtime.html#proficiency_levels[*{proficiency-foundational}*].]
+ifdef::docs-topic[xref:proficiency_advanced[*{proficiency-advanced}*].]
+ifndef::docs-topic[link:https://appdev.openshift.io/docs/wf-swarm-runtime.html#proficiency_levels[*{proficiency-advanced}*].]
 
 The Secured Booster expands on the {name-mission-http-api} Booster by securing a REST endpoint using link:https://access.redhat.com/products/red-hat-single-sign-on[{RHSSO}]. {RHSSO} implements the OAuth 2.0 specification and uses it to issue access tokens to provide clients with various access rights to secured resources. Securing an application with SSO enables you to add security to your applications while centralizing the security configuration.
+
+IMPORTANT:  You must configure {RHSSO} and Keycloak to access certain components included in the Secured Mission. This mission does not show you how to perform this configuration. Ensure that you are familiar with the basic concepts related to {RHSSO} and Keycloak before using this mission.
```
Comment (Ladicek) [inline docs/topics/secured-mission-intro-paragraph.adoc]: It's true that the mission doesn't show how to perform this configuration, but it comes with SSO preconfigured for demonstration purposes, so "You must configure {RHSSO} and Keycloak" isn't exactly accurate.

Also Keycloak is an upstream of RHSSO, so referring to "RHSSO and Keycloak" isn't exactly accurate.

Comment (inoxx03) [inline docs/topics/secured-mission-intro-paragraph.adoc]: You're right maybe just mention {RHSSO}, then.

File: docs/topics/secured-mission-intro-paragraph.adoc [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
-ifndef::docs-topic[link:https://appdev.openshift.io/docs/wf-swarm-runtime.html#proficiency_levels[*{proficiency-foundational}*].]
+ifdef::docs-topic[xref:proficiency_advanced[*{proficiency-advanced}*].]
+ifndef::docs-topic[link:https://appdev.openshift.io/docs/wf-swarm-runtime.html#proficiency_levels[*{proficiency-advanced}*].]
 
 The Secured Booster expands on the {name-mission-http-api} Booster by securing a REST endpoint using link:https://access.redhat.com/products/red-hat-single-sign-on[{RHSSO}]. {RHSSO} implements the OAuth 2.0 specification and uses it to issue access tokens to provide clients with various access rights to secured resources. Securing an application with SSO enables you to add security to your applications while centralizing the security configuration.
+
+IMPORTANT:  This  mission comes with {RHSSO} pre-configured for demonstration purposes. Accessing some of the components in this mission, for example, the web interface used for Authentication, requires changes to the default configuration.  Before using this mission, ensure that you are familiar with the basic concepts related to {RHSSO} configuration.
```
Comment (Ladicek) [inline docs/topics/secured-mission-intro-paragraph.adoc]: The provided configuration is suitable for all demonstration purposes, so "Accessing some of the components in this mission, for example, the web interface used for Authentication, requires changes to the default configuration" isn't exactly true. (Well there's a bug currently, but that's a bug, not an intention.)

I'd use something like this:

```adoc
IMPORTANT: This misson demonstrates using {RHSSO}, but doesn't explain its principles, usage or configuration.
Before using this mission, ensure that you are familiar with the concepts related to {RHSSO}.
```

Comment (inoxx03) [inline docs/topics/secured-mission-intro-paragraph.adoc]: Oh, I see. I'll work with your suggestion. Thanks:)

**Comment by inoxx03** [comment]
Amended the note and added a link to the SSO Getting started guide.

**Comment by rhoads-zach** [comment]
LGTM, merging.

**Pull request merged**